### PR TITLE
Refs #6945 - use autosign path for puppetca

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
to be merged after theforeman/puppet-foreman_proxy#350

File: config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if answers['foreman_proxy']
+  answers['foreman_proxy']['use_autosignfile'] = true
```
Comment (ekohl) [inline config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb]: This shouldn't happen unconditionally I think, only if it's nil.

File: config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  if answers['foreman_proxy'].has_key?('puppetdir')
+    puppetdir = answers['foreman_proxy']['puppetdir']
```
Comment (ekohl) [inline config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb]: Delete the answer as well?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb (hunk 1 above)
Comment (mmoll) [reply config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb]: changed.

File: config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb (hunk 2 above)
Comment (mmoll) [reply config/foreman.migrations/20170530141108_foreman_proxy_autosignfile.rb]: $puppetdir still used in `templates/puppet_proxy_legacy.yml.erb`

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
added a slash before 'autosign.conf'

**Comment by domcleal** [comment]
Thanks @mmoll.

**Pull request merged**